### PR TITLE
Dala 3851 update terms conditions

### PR DESCRIPTION
**Pull request by marcelrasbachdfine**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one test exists testing the new feature
  - [x] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] A fresh clone of dataland.com is generated (see Wiki page on "OTC" for details)
  - [ ] The feature branch is deployed to clone with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the clone server
  - [ ] The feature branch is deployed to dev1 with `Reset non-user related Docker Volumes & Re-populate` turned on, and it's verified that the CD run is green  
- [x] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server
- [x] Make sure that pull requests in the development tools repository, which are connected to this pull requests, 
are merged

### CODE REVIEW

File: dataland-frontend/src/router/index.ts [hunk 1]
```diff
@@ -127,6 +129,11 @@ const routes = [
   },
   {
     path: "/terms",
```
Comment (marcelrasbachdfine) [inline dataland-frontend/src/router/index.ts:131]: change to terms-en

Comment (ahoecherl) [reply dataland-frontend/src/router/index.ts]: Single page is now used for both. Resolving.

File: dataland-frontend/src/components/pages/TermsAndConditions.vue [hunk 2]
```diff
@@ -1,587 +1,1096 @@
...
-      </p>
+    <div class="flex justify-content-start align-items-center">
+      <img src="/public/static/icons/uk_flag.jpg" height="20" alt="English" />
+      <InputSwitch
+        data-test="terms-language-toggle-button"
+        v-model="languageGerman"
+        class="p-inputswitch-singlecolor"
+        style="margin-left: 10px; margin-right: 10px"
+      />
+      <img src="/public/static/icons/germany_flag.png" height="20" alt="German" />
```
Comment (ahoecherl) [inline dataland-frontend/src/components/pages/TermsAndConditions.vue:11]: This component is not visible in the mobile version of dataland.com That needs to be fixed.

image

Comment (AdrianGardijan) [reply dataland-frontend/src/components/pages/TermsAndConditions.vue:11]: Adjusted the margins to make the language switch stay below the header, checked it on desktop and mobile window for all screen widths

Comment (AdrianGardijan) [reply dataland-frontend/src/components/pages/TermsAndConditions.vue:11]: Adjusted the margins to make the language switch stay below the header when not signed in (as that makes the header taller), checked it on desktop and mobile window for all screen widths.
Below 290px width, the header starts looking weird and the language switch is no longer visible but I think its ok:
image

**Pull request merged**